mlabs/efficient-nft-pab/Main.hs

```diff
@@ -164,62 +156,25 @@ mkCollateral = do
     Contract.submitTxConstraintsWith @Void Hask.mempty $
       Constraints.mustPayToPubKey pkh (lovelaceValueOf 5_000_000)
 
-data CliOptions = CliOptions
-  { phk :: Hask.String
-  , authPhk :: Hask.String
-  , currencySymbol :: Hask.String
-  , tokenName :: Hask.String
-  , mintPolicy :: Hask.String
-  }
+newtype CliOptions = CliOptions {phk :: Hask.String} -- Replace with data when more params added
 
 cliOptionsParser :: Parser CliOptions
 cliOptionsParser =
   CliOptions
     Hask.<$> strOption (long "pkh" Hask.<> value "" Hask.<> help "own pub key hash")
-    Hask.<*> strOption (long "auth-pkh" Hask.<> value "" Hask.<> help "author pub key hash")
-    Hask.<*> strOption (long "currency" Hask.<> value "" Hask.<> help "currency symbol")
-    Hask.<*> strOption (long "token" Hask.<> value "" Hask.<> help "tokenName as a string")
-    Hask.<*> strOption (long "mint-policy" Hask.<> value "" Hask.<> help "arbitrary string for mintPolicy version")
 
 getCliOptions :: Hask.IO CliOptions
 getCliOptions = execParser (info (cliOptionsParser <**> helper) (fullDesc Hask.<> header "Efficient NFT PAB"))
 
 main :: Hask.IO ()
 main = do
-  CliOptions {phk, authPhk, currencySymbol, tokenName, mintPolicy} <- getCliOptions
-  let uCs = fromString currencySymbol -- CURRENCY_SYMBOL
-      auth = PaymentPubKeyHash $ fromString authPhk -- YOUR_PKH
-      mp =
-        MintParams
-          { mp'authorShare = toEnum 1000
-          , mp'daoShare = toEnum 500
-          , mp'price = toEnum 100_000_000
-          , mp'lockLockup = 5
-          , mp'lockLockupEnd = 5
-          , mp'owner = Just (auth, Nothing)
-          , mp'fakeAuthor = Just auth
-          , mp'feeVaultKeys = []
-          , mp'mintPolicy = mintPolicy
-          }
+  CliOptions {phk} <- getCliOptions
 
+  Hask.putStr "Unapplied CNFT minting policy: "
   Hask.print $ Text.decodeUtf8 . Base16.encode $ serialise policyUntypedScript
 
-  let
-    oref = TxOutRef (TxId "00e52d6fdb45e529dda8cfaa4e7f04dc8b94deffd1bd54196193cc2c5c49e418") 5
-    applied = policyUntyped oref
-  Hask.print $ PlutusTx.toBuiltinData oref
-  Hask.putStr "Applied CNFT currency symbol: "
-  Hask.print $ scriptCurrencySymbol applied
-  Hask.print $ Text.decodeUtf8 . Base16.encode $ serialise $ unMintingPolicyScript applied
-
-  -- in case of types changing, can help with request rewriting
-  -- Hask.putStr "seabug-mint-request: "
-  -- ByteString.putStrLn $
-  --   JSON.encode $
-  --     Mint (assetClass uCs $ fromString tokenName, mp)
-
-  -- Hask.putStr "unapplied-minting-policy: "
-  -- ByteString.putStrLn $ JSON.encode Token.policyDataScript
+  Hask.putStr "Unapplied sgNFT minting policy: "
```

Comment:
The other one to log is the one from Lock.hs

Reply:
Oh right, its not being logged at all right now?

Reply:
Yeah it's not being logged, I logged it, saved it then removed the log. It was logged in the same was as the cnft minting policy

Reply:
Have added now :)